UnityProject/Assets/Scripts/Player/JoinedViewer.cs

```diff
@@ -174,6 +180,12 @@ private void CmdRequestJob(JobType jobType, string jsonCharSettings)
 			return;
 		}
 
+		if (PlayerList.Instance.FindPlayerJobBanEntryServer(playerID, jobType, true) != null)
```

Comment:
I feel like this would be a bit insecure since you could send someone else's ID?

Reply:
I did originally use the SentByPlayer.Connection from the client message but it was returning some values null. It is insecure in the sense they could grab any players ban entry but it is limited to that ban info, which is just a list of this:

[Serializable]
public class JobBanEntry
{
	public JobType job;
	public bool isPerma;
	public double minutes;
	public string dateTimeOfBan;
	public string reason;
}

Reply:
but if you're banned from captain what stop you from sending someone else's ID to avoid the ban?

Reply:
The server does its own check, this is just the client asking for the data to do a client side check which while can be hacked around, the server side check would still apply.

Reply:
Its just a way to stop the client from asking the server to keep doing checks unless it knows that the check will pass.